examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
```

Comment:
lib source could be more readable IMHO but that opinion is based on just trying to do a search on `Element` to find where it's defined